### PR TITLE
Adding helm chart

### LINKED ISSUE #231: Helm Chart and Login credentials
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:

1. Deployment needs to be streamlined, this can be done easily with the help of Helm chart. If possible, can you make a helm chart for Purser.

2. Provide default login credentials and a way to disable login.

logs -

```
purser-68c74b9d9b-njwjj purser-controller ERRO[5572] rpc error: code = Unknown desc = : Attribute username is not indexed.
purser-68c74b9d9b-njwjj purser-controller ERRO[5572] rpc error: code = Unknown desc = : Attribute username is not indexed.
purser-68c74b9d9b-njwjj purser-controller INFO[5572] POST	/auth/login	Login	3.124595ms
```

**What you expected to happen**:

Default login credentials  don't work
* username - `admin`
* password - `purser!123`

found the above credentials in [search](https://github.com/vmware/purser/search?q=username&unscoped_q=username).

**How to reproduce it (as minimally and precisely as possible)**:

1. Follow the instructions in the README to deploy Purser
2. Try to login

**Anything else we need to know?**:

**Environment**:
- golang version:
- Kubernetes version (use `kubectl version`): 1.13.5
- Cloud provider or hardware configuration: AWS
- OS (e.g. from /etc/os-release): Custom
- Kernel (e.g. `uname -a`): 5.0
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by Anish-Gupta-Bose**
**What this PR does / why we need it**:
Add a helm chart to easily manage and deploy purser

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #231 

**Special notes for your reviewer**:

Please look into the RBAC rules, the current version is too open.
Please update the `README`

**Release note**:

```release-note
1. Adds Helm chart
```

### CODE REVIEW

**Comment by hemani19** [comment]
Thank you @Anish-Gupta-Bose for your contribution to Purser!

File: cluster/helm/chart/purser/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (tpepper) [inline cluster/helm/chart/purser/Chart.yaml]: Would this be better as "A Helm chart for the Purser extension to Kubernetes"?

Comment (Anish-Gupta-Bose) [reply cluster/helm/chart/purser/Chart.yaml]: Oops, might have overlooked that!

File: cluster/helm/chart/purser/templates/purser-controller-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+metadata:
+  name: {{ include "purser.fullname" . }}-controller
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "purser.name" . }}-controller
+    helm.sh/chart: {{ include "purser.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```
Comment (Steve-Louie-Bose) [inline cluster/helm/chart/purser/templates/purser-controller-deployment.yaml]: `s/.Values.replicaCount/.Values.controller.replicaCount`

**Pull request merged**